packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -1578,7 +1573,7 @@ class IntentFilterCheck {
 class Deeplink {
     String scheme, host, path
     IntentFilterCheck intentFilterCheck
-    boolean equals(o) {
+    boolean equals(Object o) {
```

Comment:
android studio linter is prompting   `Cannot determine type of 'o'`

Reply:
I actually think object o is probably better than o with no type. l